### ISSUE TITLE
1.1.1 npm install can't resolve 'datatables.net-searchPanes'

### ISSUE THREAD

**Issue by daniel-butler** [DataTables/SearchPanes#16]
After installing searchpanes with npm then testing datatables like so.
```js
  import $ from 'jquery';
  import 'datatables.net';
  // JSZip library is not accessible to data tables without it being a global variable
  import * as JSZip from 'jszip';
  window.JSZip = JSZip;

  import 'datatables.net-bs4/js/dataTables.bootstrap4.min';
  import 'datatables.net-buttons/js/buttons.html5.min';
  import 'datatables.net-buttons/js/dataTables.buttons.min';
  import 'datatables.net-buttons-bs4/js/buttons.bootstrap4.min';
  import 'datatables.net-select-bs4/js/select.bootstrap4.min';
  import 'datatables.net-searchpanes-bs4/js/searchPanes.bootstrap4.min';
```
I get the following error message

Can't resolve 'datatables.net-searchPanes' in '/app/node_modules/datatables.net-searchpanes-bs4/js'

![image](https://user-images.githubusercontent.com/22578954/91767642-7948c000-ebaa-11ea-88ee-04859482e927.png)

The versions used are
```bash
npm list --depth=0 | grep datatables
> +-- datatables.net@1.10.21
> +-- datatables.net-bs4@1.10.21
> +-- datatables.net-buttons@1.6.2
> +-- datatables.net-buttons-bs4@1.6.2
> +-- datatables.net-fixedheader@3.1.7
> +-- datatables.net-fixedheader-bs4@3.1.7
> +-- datatables.net-searchpanes@1.1.1
> +-- datatables.net-searchpanes-bs4@1.1.1
> +-- datatables.net-select@1.3.1
> +-- datatables.net-select-bs4@1.3.1
```

**Comment by SandyDatatables** [comment]
Hi @daniel-butler ,

Thanks for spotting this, it has actually already been spotted and fixed in pull request #15 . The problem is that the SearchPanes node module has a lower case "p" rather than an upper case, see the commits in the pull request. The fix will be available in the next SearchPanes release, which we hope will be very soon.

Thanks,
Sandy

**Comment by daniel-butler** [comment]
I was under the impression the fix was on 1.1.1 that's why I opened a new issue.

Thanks for all the work on this library by the way.
